osu.Game/Screens/OnlinePlay/Lounge/LoungeSubScreen.cs

```diff
@@ -98,7 +106,13 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             initialRoomsReceived.BindTo(RoomManager.InitialRoomsReceived);
-            initialRoomsReceived.BindValueChanged(onInitialRoomsReceivedChanged, true);
+            initialRoomsReceived.BindValueChanged(_ => updateLoadingLayer());
+
+            if (ongoingOperationTracker != null)
+            {
+                joiningRoom.BindTo(ongoingOperationTracker.InProgress);
```

Comment:
not sure this should be called `joiningRoom`; are you okay with renaming to `operationInProgress` in line with other usages? reasoning is that a different operation could be in progress, apart from the local one (if we only cared about the local one than checking if `joiningRoomOperation` is null would be enough).